app/controllers/review_queues_controller.rb

```diff
@@ -36,6 +48,24 @@ def submit
     render json: { status: 'ok' }
   end
 
+  def item
+    @item = ReviewItem.find(params[:item_id])
+    render 'queue.html.erb'
```

Comment:
Just `:queue` should be enough here, unless you're making requests in a different format.